configurations/es5.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    extends: [
+        '../rules/eslint/best-practices/on.js',
+        '../rules/eslint/possible-errors/on.js',
+        '../rules/eslint/ecma-script-6/off.js',
+        '../rules/eslint/node-js-and-common-js/off.js',
+        '../rules/eslint/strict-mode/on.js',
+        '../rules/eslint/stylistic-issues/on.js',
+        '../rules/eslint/variables/on.js',
+        '../rules/eslint/deprecated/off.js'
```

Comment:
To get a green light from `eslint-find-rules`. 

Currently it is throwing an error if the deprecated rules are not included.


Waiting on https://github.com/sarbbottam/eslint-find-rules/issues/172 to be resolved and then we can safely remove them